packages/ra-ui-materialui/src/list/filter/FilterButton.tsx

```diff
@@ -32,9 +33,10 @@ const FilterButton = (props: FilterButtonProps): JSX.Element => {
         showFilter,
         classes: classesOverride,
         className,
-        resource,
+        resource: propsResource,
```

Comment:
I think you can just remove the line

Reply:
But the `resource` prop is going to be injected into the div at line 76